### PR TITLE
.mailmap and .orgmap: organization affiliation

### PR DESCRIPTION
**Pull request by jjhuo**
Signed-off-by: Jianjian Huo <jianjian.huo@ssi.samsung.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
LGTM

File: .organizationmap [hunk 1]
```diff
@@ -339,6 +339,7 @@ Roald van Loon Consultancy <roald@roaldvanloon.nl> Roald J. van Loon <roald@roal
 Ruijie Networks <contact@ruijie.com.cn> Ning Yao <yaoning@ruijie.com.cn>
 Sandia National Laboratories <contact@sandia.gov> Jim Schutt <jaschut@sandia.gov>
 Samsung <contact@samsung.com> James Liu <james.liu@ssi.samsung.com>
+Samsung <contact@samsung.com> Jianjian Huo <samuel.huo@gmail.com>
```
Comment (theanalyst) [inline .organizationmap]: here it should be `Samsung <contact@samsung.com> <jianjian.huo@ssi.samsung.com>`, the mailmap will automatically associate your work email as primary

**Comment by jjhuo** [comment]
Done, thanks!

File: .organizationmap [hunk 2]
```diff
@@ -339,6 +339,7 @@ Roald van Loon Consultancy <roald@roaldvanloon.nl> Roald J. van Loon <roald@roal
 Ruijie Networks <contact@ruijie.com.cn> Ning Yao <yaoning@ruijie.com.cn>
 Sandia National Laboratories <contact@sandia.gov> Jim Schutt <jaschut@sandia.gov>
 Samsung <contact@samsung.com> James Liu <james.liu@ssi.samsung.com>
+Samsung <contact@samsung.com> <jianjian.huo@ssi.samsung.com> Jianjian Huo <samuel.huo@gmail.com>
```
Comment (theanalyst) [inline .organizationmap]: I'm sorry I wasn't very clear last time, basically this line only needs the preferred (ie work) email.. so 
`Samsung <contact@samsung.com> Jianjian Huo <jianjian.huo@ssi.samsung.com> ` 
would be the right entry.. as seen for other entries etc.

**Comment by jjhuo** [comment]
I see. had that fixed, thanks again!

**Comment by theanalyst** [comment]
LGTM

**Comment by jjhuo** [comment]
@theanalyst could you please merge it? thanks!

**Pull request merged**